CHANGELOG.md

```diff
@@ -1,7 +1,20 @@
 # Change Log (Release Notes)
 
 All _notable_ changes to this project will be documented in this file (`CHANGELOG.md`).
-This project adheres to [the Python form of Semantic Versioning](https://packaging.python.org/tutorials/distributing-packages/#choosing-a-versioning-scheme) (or at least we try).
+
+This project adheres to [Semantic Versioning 2](http://semver.org/spec/v2.0.0.html), the BigchainDB **public API**:
+* Includes:
+  * This definition of **public API**.
+  * The Command Line Interface.
+  * The transaction schema.
+  * All the endpoints exposed by the HTTP API (including payloads sent to and received from them).
+  * The docker setup.
+* Excludes:
```

Comment:
> As we see transactions and assets as the main interaction paradigm of BigchainDB, we could view blocks and votes as external, hence not give backwards compatibility guarantees regarding semver.

IMO changing those models breaks compatibility:
* If someone is building a *bigchaindb explorer* they will use those models.
* Changing those models means to migrate the DB and—I guess—all the nodes should "vote again".

If this is what we want, then we might explicitly add those models to this PR, even if they are included in the sentence:
> All the endpoints exposed by the HTTP API (including payloads sent to and received from them)